homeassistant/components/comelit/cover.py

```diff
@@ -99,3 +110,18 @@ async def async_stop_cover(self, **_kwargs: Any) -> None:
 
         action = COVER_OPEN if self.is_closing else COVER_CLOSE
         await self._api.cover_move(self._device.index, action)
+
+    @callback
+    def _handle_coordinator_update(self) -> None:
+        """Handle device update."""
+        self._last_state = self.state
+        self.async_write_ha_state()
+
+    async def async_added_to_hass(self) -> None:
+        """Handle entity which will be added."""
+        _LOGGER.info("Restoring entity %s", self._device.name)
```

Comment:
```suggestion
```

Reply:
It should have been debug, but maybe it's redundant anyway. Removed